### PR TITLE
New runtime upgrade

### PR DESCRIPTION
**Pull request by alexkdefi**
## Issue
https://github.com/ComposableFi/SRE/issues/76
## Description
GA for runtime upgrade 

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

File: .github/workflows/new-runtime-upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+name: "NodeJs runtime upgrade"
```
Comment (KaiserKarel) [inline .github/workflows/new-runtime-upgrade.yml:1]: Are we upgrading Nodejs?

File: .github/workflows/new-runtime-upgrade.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+      - name: Install
+        working-directory: scripts/github/
+        run: yarn install
+
+      - name: Upgrade runtime
+        working-directory: scripts/github/
+        run: |
+          FILE_NAME=picasso_runtime.wasm
+          yarn upgrade-runtime --root_key=${{ secrets.PICASSO_ROOT_KEY }} --rpc_ws_url=wss://rpc.composablefinance.ninja --path=./${FILE_NAME}
```
Comment (KaiserKarel) [inline .github/workflows/new-runtime-upgrade.yml:141]: FYI, once we launch, we will no longer have access to `root`. These 2 actions would then need to become: submit runtime upgrade to chain for proposal.

**Review by KaiserKarel — CHANGES_REQUESTED** [review]
Please squash your commits, to remove the unnecessary test-run messages.

File: .github/workflows/new-runtime-upgrade.yml (hunk 2 above)
Comment (ch00mba) [reply .github/workflows/new-runtime-upgrade.yml:141]: ok. That would be multisig right ? Which is basically another extrinsic. Correct ? @KaiserKarel

Comment (alexkdefi) [reply .github/workflows/new-runtime-upgrade.yml:141]: ok

**Comment by alexkdefi** [comment]
Bors do squash

**Pull request closed without merging**